pkgs/applications/video/epgstation/update.nix

```diff
@@ -49,18 +55,8 @@ in writers.writeBash "update-epgstation" ''
 
   # Generate default streaming settings for the nixos module.
   pushd ../../../../nixos/modules/services/video/epgstation
-  ${jq}/bin/jq '
-    { liveHLS
-    , liveMP4
-    , liveWebM
-    , mpegTsStreaming
-    , mpegTsViewer
-    , recordedDownloader
-    , recordedStreaming
-    , recordedHLS
-    , recordedViewer
-    }' \
-    "$SRC/config/config.sample.json" \
+  ${yq}/bin/yq -j '{ urlscheme , stream }' \
+    "$SRC/config/config.yml.template" \
     > streaming.json
```

Comment:
I'll rephrase: why can't a newline be added to the file automatically after this step?

Reply:
I've now removed the EditorConfig exception and did that instead. I didn't do it initially because I was concerned that there'll be other ways for EditorConfig checks to fail in future updates.